core/domain/state_domain.py

```diff
@@ -973,6 +2074,46 @@ def validate(
             raise utils.ValidationError(
                 'Hint(s) must be specified if solution is specified')
 
+        # These validations should only be run when we are validating for
+        # an exploration because as of now(16 Sept 2022) the data related
+        # to Question model is not yet fixed.
+        if validation_from_exploration:
+            # Validation for Continue interaction.
```

Comment:
you can remove these one-line comments and the new lines since it's self-explanatory.